docs/applications/configuration-management/Using-Laravel-Forge-with-Linode.md

```diff
@@ -0,0 +1,226 @@
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+alias: [  ]
+modified: Tuesday, August 8th, 2017 
+modified_by:
+    name: Onwuka Gideon
+published: ' '
+title: 'Using Laravel Forge with Linode'
+external_resources:
+ - '[Laravel Forge](https://forge.laravel.com)'
+---
+
+
+[Laravel Forge](https://forge.laravel.com) is not all about the [Laravel framework](https://laravel.com). Forge is an incredible tool that provides a GUI interface for interacting with servers seamlessly. It’s in the family of Laravel products that help developers focus on developing and not deploying and hosting. You can deploy any of your PHP related project easily on Linode with the help of Forge which provides a GUI feature for managing and running commands on your Linode server.
+
+ By default, when deploying our project, we usually install Nginx, PHPMyAdmin, PHP and many other things needed. Laravel Forge does all the necessary installation and configuration for you, you just need to do a little configuration and your website will be up.
+
+ This guide covers how to use Laravel Forge with Linode.
+ 
+
+## Before You Begin
+
+1. Sign up for a Larvel Forge account if you don't have one ([Laravel Forge](https://forge.laravel.com/auth/register))
+2. Sign up or Log into your Linode account. ([Linode](https://www.linode.com/))
+
+## Link to a Source Control (GitHub, Gitlab or Bitbucket)
+
+If you want to deploy from either of these services( Github, Gitlab or bitbucket), you need to link them to your Forge account. Once you are logged in to your Forge Account, head to your profile tab.
+
+1. Choose your preferred source control by clicking on the `Source Control` Link on the Left.
+
+[![Linode API](/docs/assets/configuration-management/linking_source_control.png)](/docs/assets/configuration-management/linking_source_control.png)
+
+2. **Click** on either; `CONNECT TO GITHUB` , `CONNECT TO GITLAB`, or `CONNECT TO BITBUCKET`. In this tutorial I'll Gitlab, So I clicked on `CONNECT TO GITLAB`. 
+
+Once you select the Source Control you want, you’ll be redirected to the Source control website where you need to authorize Laravel Forge and then you will be redirected back to Forge’s website.
+
+[![Linode API](/docs/assets/configuration-management/source_control_authorized.png)](/docs/assets/configuration-management/source_control_authorized.png)
+
+
+## Add SSH keys to your profile.
+You should have received an email already about details of your ssh from above. 
+
+[![Linode API](/docs/assets/configuration-management/adding_public_key.png)](/docs/assets/configuration-management/adding_public_key.png)
+
+Click on the `SSH Keys` and upload your local SSH Key by following the link on the mail or see How to create and copy an SSH Key.  Once you have your SSH key, add a name and your key to the form and submit by clicking on `ADD KEY`
+
+
+## Get your Linode API key
```

Comment:
Remove this step. Just link existing Linode guide: https://www.linode.com/docs/platform/api/api-key